packages/wrangler/src/config/index.ts

```diff
@@ -507,7 +535,7 @@ export function printBindings(bindings: CfWorkerInit["bindings"]) {
 
 	if (unsafe?.metadata !== undefined) {
 		output.push({
-			type: "Unsafe Metadata",
+			name: friendlyBindingNames.unsafe,
```

Comment:
This has been changed from "Unsafe Metadata" to "Unsafe" – was that intentional?